contracts/signature-validators/OffChainSignatureValidator.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.20;
+
+import { IERC1271 } from "openzeppelin/contracts/interfaces/IERC1271.sol";
+import { Strings } from "openzeppelin/contracts/utils/Strings.sol";
+import { Party } from "../party/Party.sol";
+
+/// @notice Contract that by default validates off-chain signature for parties
+contract OffChainSignatureValidator is IERC1271 {
+    error NotMemberOfParty();
+    error InsufficientVotingPower();
+    error MessageHashMismatch();
+
+    event SigningThresholdBipsSet(
+        Party indexed party,
+        uint96 oldThresholdBips,
+        uint96 newThresholdBips
+    );
+
+    mapping(Party party => uint96 thresholdBips) public signingThersholdBips;
```

Comment:
We've usally referred to it as "bps" instead of "bips"

```suggestion
    mapping(Party party => uint96 thresholdBps) public signingThersholdBips;
```

Reply:
👍